src/cryptography/hazmat/decrepit/ciphers/algorithms.py

```diff
@@ -0,0 +1,25 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import annotations
+
+from cryptography.hazmat.primitives.ciphers.algorithms import (
+    _BlowfishInternal as Blowfish,
+)
+from cryptography.hazmat.primitives.ciphers.algorithms import (
+    _CAST5Internal as CAST5,
+)
+from cryptography.hazmat.primitives.ciphers.algorithms import (
+    _IDEAInternal as IDEA,
+)
+from cryptography.hazmat.primitives.ciphers.algorithms import (
+    _SEEDInternal as SEED,
+)
```

Comment:
Surely we should just do the reverse -- move the classes here, and re-export them in the other place but deprecated.

Reply:
Circular dependencies make this annoying, but we can do it (albeit with a significantly larger diff probably). We could also do that as a followup PR.

Reply:
(done)